src/ctap/storage.rs

```diff
@@ -56,9 +56,14 @@ const GLOBAL_SIGNATURE_COUNTER: usize = 1;
 const MASTER_KEYS: usize = 2;
 const PIN_HASH: usize = 3;
 const PIN_RETRIES: usize = 4;
-const NUM_TAGS: usize = 5;
+const ATTESTATION_PRIVATE_KEY: usize = 5;
+const ATTESTATION_CERTIFICATE: usize = 6;
+const AAGUID: usize = 7;
+const NUM_TAGS: usize = 8;
 
 const MAX_PIN_RETRIES: u8 = 6;
+const ATTESTATION_PRIVATE_KEY_LENGTH: usize = 32;
```

Comment:
This constant is from the crypto library in ec/int256.rs called BITS_PER_DIGIT. Since you also use this to double-check, maybe it's intended to be its own constant?

Reply:
I'm not sure what you mean. The BITS_PER_DIGIT constant is private so I can't use it here. Do you mean to make it public?

The reason I added the constant here was that it is hard-coded in the key_material.rs file. And I need it multiple times in this file so I just factored it out. I'm fine using a shared constant if there's one.

Reply:
Yeah, I meant making it public. Not sure if just having your own constant isn't better though. So feel free to keep it like that, just wanted to pointed out.

Reply:
If it makes sense for the constant (actually it looks like it should be `NBYTES` and not `BITS_PER_DIGIT` according to the signature of `Int256::from_bin`) to be part of the crypto public API, then I should use it here. As I understand it, it's OK to make it public so I did it and used the constant here.

Reply:
Yeah sorry, definitely NBYTES. LGTM